src/main/java/org/apache/maven/shared/verifier/Verifier.java

```diff
@@ -210,7 +211,7 @@ private Verifier( String basedir, String settingsFile, boolean debug, Boolean fo
             forkMode = "auto";
         }
 
-        this.defaultCliOptions = defaultCliOptions == null ? new String[0] : defaultCliOptions.clone();
+        this.defaultCliArguments = defaultCliArguments == null ? new String[0] : defaultCliArguments.clone();
```

Comment:
Stupid question: How does this compare to ` Arrays#copyOf()`?

Reply:
There is no method ... `Arrays#copyOf(String [])`

`clone` on array of String will do shadow clone - it will ok for primitive object and also for String, String instance is immutable.

I didn't think in that detail on this code ... 😄